src/Aspire.ProjectTemplates/templates/aspire-starter/AspireStarterApplication.1.Tests/WebTests.cs

```diff
@@ -0,0 +1,22 @@
+using System.Net;
+
+namespace AspireStarterApplication.1.Tests;
+
+public class WebTests
```

Comment:
Is `WebTests` the best name? `DistributedApplicationTests` maybe? Or use `IntegrationTest` like in the other template?

Reply:
I named it that after the "Web" project in that template.

Reply:
Ah, makes sense.